README.rst

```diff
@@ -44,11 +48,24 @@ Features
 - support for several shared storage systems (Ceph)
 - support for several container technologies (Docker)
 
+Getting started
+---------------
+
+You can `install REANA locally <http://docs.reana.io/development/deploying-locally/>`_, `deploy it at scale on premises
+<http://docs.reana.io/development/deploying-at-scale/>`_ (in about 10 minutes) or use https://reana.cern.ch. Once the system
+is ready, you can follow the guide to run `your first example <http://docs.reana.io/getting-started/first-example/>`_.
+For more in depth information visit the `official REANA documentation <http://docs.reana.io/>`_.
+
+Community
+---------
+
+- Discuss `on Forum <https://forum.reana.io/>`_
+- Chat `on Gitter <https://gitter.im/reanahub/reana>`_
```

Comment:
We could also advertise Mattermost here, as in `chat on _Mattermost_ or _Gitter_`.